test/NinjaTools.FluentMockServer.API.Exclude.Tests/Downstream/RequestMapperSpec.cs

```diff
@@ -112,19 +105,13 @@ private async Task ThenTheMappedContentIs([NotNull] HttpContent expected)
             var actualData = await _result.Data.Content.ReadAsStringAsync();
             var expectedData = await expected.ReadAsStringAsync();
 
-            if (actualData != null)
-            {
-                Output.WriteLine($"Actual data:\n{actualData}");
-            }
+            if (actualData != null) Output.WriteLine($"Actual data:\n{actualData}");
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Add curly braces around the nested statement(s) in this 'if' block.](https://app.codacy.com/manual/git_36/NinjaTools.FluentMockServer/pullRequest?prid=4739715)